karaf/roboconf-karaf-dist-agent/pom.xml

```diff
@@ -84,7 +78,23 @@
 				<artifactId>karaf-maven-plugin</artifactId>
 				<configuration>
 					<bootFeatures>
-						<feature>standard</feature>
+						<!-- Karaf 4.x has split the "standard" features into many sub-features -->
+						<feature>shell-compat</feature>
```

Comment:
For sure we don't! I've just replaced the standard feature with all its content, to make things work quickly ;)
Maybe another issue would be to lighten the distributions, removing all the unnecessary features...